pkg/controller/secret/secret_controller.go

```diff
@@ -268,6 +271,10 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 
 		// https://issues.redhat.com/browse/OSD-10485
 		{Receiver: receiverMakeItWarning, Match: map[string]string{"alertname": "etcdHighNumberOfFailedGRPCRequests", "namespace": "openshift-etcd"}},
+
+		// https://issues.redhat.com/browse/OSD-11298
+		{Receiver: receiverMakeItCritical, MatchRE: map[string]string{"name": ".*master.*"}, Match: map[string]string{"alertname": "MachineWithoutValidNode", "namespace": "openshift-machine-api"}},
+		{Receiver: receiverMakeItCritical, MatchRE: map[string]string{"name": ".*master.*"}, Match: map[string]string{"alertname": "MachineWithNoRunningPhase", "namespace": "openshift-machine-api"}},
```

Comment:
could match more specifically if we are worried about some node names like "mastercard-worker-1":
```
.*-master-[123]
```

Reply:
+1 - lets be as precise as possible with regex. something like `.+-master-[123]` ?

Reply:
changed to `^.+-master-[123]$` and added a comment with link to regex test cases: https://regex101.com/r/Rn6F5A/1